dashboard/src/scenes/action/list.js

```diff
@@ -100,7 +101,7 @@ const List = () => {
             <SelectStatus noTitle onChange={(event) => setStatus(event.target.value)} value={status} />
           </div>
         </Col>
-        <Col md={12} style={{ display: 'flex', alignItems: 'center', marginBottom: 20 }}>
+        <Col md={6} style={{ display: 'flex', alignItems: 'center', marginBottom: 20 }}>
```

Comment:
Pour éviter ça, il vaut mieux garder comme avant :

```suggestion
        <Col md={12} style={{ display: 'flex', alignItems: 'center', marginBottom: 20 }}>
```

<img width="1012" alt="Capture d’écran 2022-03-21 à 10 42 04" src="https://user-images.githubusercontent.com/1575946/159236742-c5bcec79-3d87-4f3f-81c9-6d5f92b646e0.png">